bouncer/shared/utils.ts

```diff
@@ -951,6 +951,26 @@ export async function submitChainflipExtrinsic(
   return extrinsicResult;
 }
 
+export class ChainflipExtrinsicSubmitter {
+  private keyringPair: KeyringPair;
+
+  private mutex: Mutex;
+
+  constructor(keyringPair: KeyringPair, mutex: Mutex = new Mutex()) {
+    this.keyringPair = keyringPair;
+    this.mutex = mutex;
+  }
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  async Submit(extrinsic: any, errorOnFail: boolean = true) {
```

Comment:
Why is this capitalised? I don't think we use this style for any methods (both in js and rust).

Reply:
woops. strange that prettier or something did complain about this.